app/package/models.py

```diff
@@ -335,7 +340,8 @@ class MetaDataDB(db.Model):
     name = db.Column(db.TEXT, index=True)
     version = db.Column(db.TEXT, index=True, default='latest')
     descriptor = db.Column(db.JSON)
-    status = db.Column(db.TEXT, index=True, default='active')
+    status = db.Column(db.Enum(PackageStateEnum, native_enum=False),
+                       index=True, default=PackageStateEnum.active)
```

Comment:
In our case - we will almost never face a query for "all deleted packages". As far as I understand - only this query *may* have some benefit for the index, as there would likely be only few deleted packages. But our most common case will be to get all the active packages (plus some publisher filter). If the optimizer decides to use index here - it will lekely to slow down the query (compared to fullscan), so we have to avoid it. And overall the write costs of maintaining such an index will outweight its benefits.